Advebture game - Search methods

janetheboss opened the pull request:
Added 2 new methods to PlayerService to search a single player and an…other
Added new seeder class called PlayerData
And updated unite test to contain the new unit test for the search methods

LexterLab commented on src/Main.java:
```diff
@@ -6,27 +6,86 @@ public static void main(String[] args) {
-        System.out.println("----- Create Player -----");
-        playerService.createPlayerIfUserExists(userService.getRegisteredUser());
+            switch (choice) {
+                case 1:
```
Try using modern switch cases with -> lambda instead

janetheboss committed:
Added 2 new methods to PlayerService to search a single player and another
Added new seeder class called PlayerData
And updated unite test to contain the new unit test for the search methods

janetheboss committed:
added new class for the display of the menu and made it using lambda

janetheboss committed:
made the switch case modern

janetheboss committed:
made the switch case modern like java 17
